Gemfile.lock

```diff
@@ -127,14 +131,14 @@ GEM
       concurrent-ruby (~> 1.0)
       dry-core (~> 0.2)
       dry-equalizer (~> 0.2)
-    dry-schema (1.3.4)
+    dry-schema (1.4.1)
```

Comment:
Was this necessary for the Rails 5.2.3 upgrade?

Upgrading dry-* isn't a problem, but it mixes Rails upgrade with other upgrades.

Reply:
After changing the rails version to 5.2.3 I ran a `bundle update` to bring in the newer gem